visualc/VS2010/aescrypt2.vcxproj

```diff
@@ -95,7 +95,7 @@
       <Optimization>Disabled</Optimization>
       <PreprocessorDefinitions>WIN32;_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <AdditionalIncludeDirectories>
-../../include;../../crypto/include
+../../include;../../crypto/include;../../crypto/3rdparty/everest/include/;../../crypto/3rdparty/everest/include/everest;../../crypto/3rdparty/everest/include/everest/vs2010;../../crypto/3rdparty/everest/include/everest/kremlib

```

Comment:
Which Windows tests run with `scripts/config.pl set MBEDTLS_ECDH_VARIANT_EVEREST_ENABLED`? Do we have any tests that MBEDTLS_ECDH_VARIANT_EVEREST_ENABLED works on Windows?

Reply:
No we don't. Currently we are testing Windows builds only with the default configuration in `all.sh` with a mingw cross build and we don't run it. In the test system we build and run a modified full build, but Everest is excluded from the full build.

We could add testing for Everest on windows, but the best place for that would be in the test repo and I don't think it should block this PR. I have raised a ticket to track the issue.

Reply:
Note that we are tracking this test deficiency internally via IOTSSL-2900